modules/nf-core/rtn/tna/tests/main.nf.test

```diff
@@ -0,0 +1,70 @@
+nextflow_process {
+
+    name "Test Process RTN_TNA"
+    script "../main.nf"
+    process "RTN_TNA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rtn"
+    tag "rtn/tni"
+    tag "rtn/tna"
+
+    setup {
+        def content = "CYP24A1\nNNAT\nPHACTR3"
+        def DEGs = file("${workDir}/degs.tsv")
+        DEGs.text = content
+
+        run("RTN_TNI") {
```

Comment:
You can also save the output of the `rtn/tni` package on https://github.com/nf-core/test-datasets/, to avoid running it twice (since it is already ran in its own test suite).